adding_species_manuscript.tex

```diff
@@ -279,7 +279,7 @@ \section*{The utility of \stdpopsim for genome-wide simulations}
 see \citet{Adrion2020} for more on the topic.
 The main objective of population genomic simulations is to recreate 
 patterns of sequence variation along the genome under known conditions
-that model a given species (or population) of interest.
+that are modeled in a given species (or population) of interest.
```

Comment:
Perhaps "reflect a given species"?

Reply:
All options sound equally good to me. I guess that if I have to say where the problem in this sentence is it is in the 'conditions'. What are these known conditions that we're referring to here? Maybe a more accurate description would be 'under the assumed evolutionary history of a given species'?

Reply:
I think that's better, yes. The issue I see is that the sentence otherwise risks sounding redundant (even if it's not), as in "The main objective is to recreate patterns of sequence variation... that look like the variation in a given species." Your suggestion fixes that.